policy_relay.yaml

```diff
@@ -0,0 +1,104 @@
+enqueue:
+
+  announceBlockHeader:
+    - all:
+        nodeType: 'core'
+        maxAhead: 0
+        precedence: 'highest'
+    - all:
+        nodeType: 'relay'
+        maxAhead: 0
+        precedence: 'highest'
+    - all:
+        nodeType: 'edge'
+        maxAhead: 0
+        precedence: 'medium'
+
+  requestBlockHeaders:
+    - all:
+        nodeType: 'core'
+        maxAhead: 2
+        precedence: 'high'
+    - all:
+        nodeType: 'relay'
+        maxAhead: 2
+        precedence: 'high'
+
+  requestBlocks:
+    - one:
+        nodeTypes: ['relay', 'core']
```

Comment:
Shouldn't this be ['relay', 'core', 'edge']?

Reply:
Good idea.

Reply:
The reason I didn't bother is that the current yaml parser for policies can't express different enqueue policies for `MsgRequestBlockHeaders` depending on whether or not we have a particular set of `NodeId`s to restrict it to.